include/ignition/gazebo/gui/Gui.hh

```diff
@@ -19,6 +19,7 @@
 #define IGNITION_GAZEBO_GUI_GUI_HH_
 
 #include <memory>
+#include <string>
```

Comment:
Is this include needed?

Reply:
I didn't have it initially but `codecheck` was not happy